### PR TITLE
Replacing `find_node()` with `get_node()` as `find_node()` doesn't exist

### LINKED ISSUE #8218: Example in running_code_in_the_editor page uses "find_node()" function. This function has been removed.
**Your Godot version:**
4.1

**Issue description:**
Example in page uses "find_node()" function. This function has been removed.

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/tutorials/plugins/running_code_in_the_editor.html

Edit: I had accidentally written "get_node" instead of "find_node"

### PR DESCRIPTION
**Pull request by k0T0z**
Fixes #8218 

https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-get-node

### CODE REVIEW

File: tutorials/scripting/c_sharp/c_sharp_style_guide.rst [hunk 1]
```diff
@@ -262,7 +262,7 @@ Rather than:
 
 .. code-block:: csharp
 
-    FindNode()?.Change(wpnDmg);
+    GetNode()?.Change(wpnDmg);
```
Comment (AThousandShips) [inline tutorials/scripting/c_sharp/c_sharp_style_guide.rst:265]: This isn't necessarily correct, this isn't the same method as it has no arguments provided

Comment (k0T0z) [reply tutorials/scripting/c_sharp/c_sharp_style_guide.rst]: Correct

**Review by skyace65 — APPROVED** [review]
Looks good, this change doesn't need to be made on any other pages.

**Comment by skyace65** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1.

**Pull request merged**